.github/README.adoc

```diff
@@ -0,0 +1,27 @@
+= Action usage
+
+
+=== Build action
+
+On each push/pull_request on master and dev branch, *build* is trigger
+
+=== Which workflow for which result ?
+
+|===
+|Workflow name|description
+|Release| Create github release and could call `Publish binaries` workflow
+|Publish binaries|Push binaries on repository (maven central), as asset on github release and Could call `Open Studio PR` workflow
+|Open Studio PR|Open a PR on studio repository with new uid release version
+|===
+
+=== How to do a complete release ?
+
+To do a release, run *Release* action. You need to enter any parameters
```

Comment:
"You need to enter any parameters" => "All the parameters are optional"

Reply:
Nop ! All parameters are mandatory